docs/RELEASING.md

```diff
@@ -1,33 +1,53 @@
 # Releasing the Connectors project
 
-The version scheme we use is **MAJOR.MINOR.PATCH.BUILD** and stored in the [VERSION](https://github.com/elastic/connectors/blob/main/connectors/VERSION) file at the root of this repository.
+In versions 8.15 and earlier, the versioning scheme we used was **MAJOR.MINOR.PATCH.BUILD**.
 
-## Unified release
+Starting in 8.16, we began using semantic versioning aligned with the rest of the Elastic stack (**MAJOR.MINOR.PATCH**).
 
-**MAJOR.MINOR.PATCH** should match the Elastic and Enterprise Search version it targets and the *BUILD* number should be set to **0** the day the Connectors release is created to be included with the Enterprise Search distribution.
-
-For example, when shipping for `8.1.2`, the version is `8.1.2.0`.
-
-To release Connectors:
-
-1. Make sure all tests and linter pass with `make lint test`
-2. Run `make release`
-3. Set the [VERSION](../connectors/VERSION) file to the new/incremented version on the release branch
-4. PR this change to the appropriate Connectors release branch
-
-A package will be generated in `dist/`
+The version is stored in the [VERSION](https://github.com/elastic/connectors/blob/main/connectors/VERSION) file at the root of this repository.
 
+## After Feature Freeze
 Take care of the branching (minor releases only):
 
 - Increment the VERSION on main to match the next minor release
 - Create a new maintenance branch
 - Make sure the `.backportrc.json` is updated. The previous minor is added to `targetBranchChoices` and the new minor is used in `branchLabelMapping`
 
+
+## Unified release, (>= 8.16)
+
+The VERSION file should match that of the Elastic stack.
+
+On the day of the Feature Freeze, `#mission-control` will notifiy the release manager that it's time to create a new maintenance branch.
+
+On the day of the release, `#mission-control` will notify the releae manager that it's time to bump the VERSION to the next PATCH.
+
+The Unified Release build will take care of producing git tags and official artifacts from our most recent DRA artifacts.
+
+### In-Between releases
+
+Sometimes, we need to release Connectors independently from the Elastic unified-release.
+For instance, if a user reports a critical bug in Connector Clients, and we want to ship a fix to them as soon as possible.
+
+In this case, we can trigger the Agent Release Pipeline (once the bugfix has been merged and a staging DRA artifact produced).
```

Comment:
> we can trigger the Agent Release Pipeline

I think we should add more details how to actually trigger agent release pipeline, link perhaps? hould Let's have a todo issue to update it once we iron out the details?

Reply:
I've added links, and also which slack channel is the right one. I'd originally been under the impression that we could trigger it ourselves without any oversight, but it seems that the processes/gates are still being discussed for this release process.